packages/TokenRunner/src/Configuration/Configuration.php

```diff
@@ -2,6 +2,9 @@
 
 namespace Symplify\TokenRunner\Configuration;
 
+/**
+ * @todo here could be added break: true, inline: true options asked by @enumag
+ */
 final class Configuration
```

Comment:
@enumag Do you remember? :)

Reply:
I do. :) But as I said before I don't think it's the best solution. I don't really want to disable break or inline I just have different opinion about when and how the lines should be broken. (Mainly the rule that if a call is multiline because of an array or closure then each parameter should start on its own line.)

Reply:
Well, it's up to you, what you prefer :)

What is important, that now you can send PR to 1 place and add configuration easily. Before this it was spread in ~8 classes and adding such feature would be suicide :dagger: